anomaly-detection/build.gradle

```diff
@@ -19,13 +19,12 @@ apply plugin: 'application'
 
 sourceCompatibility = "1.8"
 mainClassName = 'io.pravega.anomalydetection.ApplicationMain'
-applicationDefaultJvmArgs = ["-Dlogging.config=file:conf/logback.xml"]
```

Comment:
Earlier you commented out the logback dependency.  Correspondingly I observed that the logback file was totally ineffectual.    There's a log4j dependency creeping in anyway due to elasticsearch.   Switching to log4j solved all issues.